tests/wsl/firstrun.pm

```diff
@@ -8,17 +8,23 @@
 # without any warranty.
 
 # Summary: Configure WSL users
-# Maintainer: Martin Loviska <mloviska@suse.com>
+# Maintainer: qa-c  <qa-c@suse.de>
 
-use base "windowsbasetest";
-use strict;
-use warnings;
+use Mojo::Base qw(windowsbasetest);
 use testapi;
-use version_utils 'is_sle';
+use version_utils qw(is_sle);
+
+sub is_sut_dev {
+    return get_var('BETA', 0);
+}
+
+sub is_sut_reg {
```

Comment:
Same here. `is_registered`?

Reply:
if it is registration eligible SUT

Reply:
`is_registrable` :D 
Just kidding, it's ok as it is :)